README.md

```diff
@@ -1,47 +1,111 @@
 [![GoDoc](https://godoc.org/github.com/timescale/timescale-prometheus?status.svg)](https://pkg.go.dev/github.com/timescale/timescale-prometheus)
 
-With this remote storage connector, Prometheus can use TimescaleDB as a compressed, long-term store for time-series metrics.
-For a detailed description of this architecture, please see [our design doc][design-doc].
+This project connects Prometheus to TimescaleDB, creating an
+**analytical platform and long-term storage for Prometheus metrics**.
 
-# Quick start with Helm (recommended)
+<img src="./docs/timescale-prometheus-arch.png>
+
+This platform is **horizontally-scalable**, **highly-compressed**, and
+**operationally mature**, allowing a user
+to ask any question, create any dashboard, and achieve greater
+visibility into their metrics.
+
+For a detailed description of this architecture, please see [our design
+doc][design-doc].
+
+### About TimescaleDB
+
+**[TimescaleDB](https://github.com/timescale/timescaledb)** is a
+**distributed time-series database built on PostgreSQL** that scales to
+over 10 million of metrics per second, supports native compression,
+handles high cardinality, and offers native time-series capabilities,
+such as data retention policies, continuous aggregate views,
+downsampling, data gap-filling and interpolation.
+
+TimescaleDB also supports full SQL, a variety of data types (numerics,
+text, arrays, JSON, booleans), and ACID semantics. Operationally mature
+capabilities include high-availability, streaming backups, upgrades over
+time, roles and permissions, and security.
+
+TimescaleDB has a **large and active user community** (tens of millions
+of downloads, hundreds of thousands of active deployments, Slack channel
+with 4,000+ members). It is natively supported by Grafana via the
+PostgreSQL/TimescaleDB datasource. Users include Siemens, Schneider
+Electric, Warner Music, Fujitsu, Comcast, and thousands of others.
+
+TimescaleDB is already the main metrics database behind one of the
+largest monitoring providers in the world, and is the preferred
+time-series database backend natively supported by Zabbix.
+
+## Getting started with Helm (recommended)
 
 The quickest way to get started is by using the Helm chart from
 [timescale-observability](https://github.com/timescale/timescale-observability),
-which provides a broader framework for installing an end-to-end monitoring
-stack.
+which provides a broader framework for installing an end-to-end
+monitoring stack.
 
-The following command will install Prometheus, TimescaleDB, Timescale-Prometheus Connector, and Grafana
+The following command will install Prometheus, TimescaleDB,
+Timescale-Prometheus Connector, and Grafana
 into your Kubernetes cluster:
 ```
 helm repo add timescale https://charts.timescale.com/
 helm repo update
 helm install --devel <release_name> timescale/timescale-observability
 ```
 
-# Configuring Helm Chart
+## Getting started (non-Helm)
```

Comment:
I think all the non-helm stuff belongs in advanced. It's also a bit weird to discuss configuring non-helm installation before discussing how to install  them.

Reply:
I think the old organization is more organic

Reply:
If you want a section here for non-helm install I would just describe the steps for non-helm installs and link to relevant advanced sections. Something like:
1) Install and run TimescaleDB (link to tsdb github repo)
2) Install (link), configure (link) and run the connector.
3) Run the cron job for the connector (link) 
4) Configure the Prometheus server to write to the connector (link).
5) Run Prometheus

Reply:
Updated by just moving non-Helm section back to Advanced